packages/mattermost-redux/src/selectors/entities/preferences.ts

```diff
@@ -214,3 +214,7 @@ export function isCustomGroupsEnabled(state: GlobalState): boolean {
 export function getUseCaseOnboarding(state: GlobalState): boolean {
     return getFeatureFlagValue(state, 'UseCaseOnboarding') === 'true';
 }
+
+export function isInsightsEnabled(state: GlobalState): boolean {
```

Comment:
I think boolean functions should be named as true/false statements rather than yes/no questions:
```suggestion
export function insightsAreEnabled(state: GlobalState): boolean {
```